src/Castle.Facilities.AspNetCore.Tests/AspNetCoreFacilityTestCase.cs

```diff
@@ -76,7 +76,7 @@ public void Should_throw_if_framework_components_registered_in_Windsor_to_preven
 			Assert.Throws<Exception>(() =>
 			{
 				container.Register(Component.For<FakeFrameworkComponent>());
-				container.ValidateConfiguration();
+				container.NoAspNetCoreRegistrations();
```

Comment:
Not sure if I'm being picky, but this sounds like it turns on a flag, maybe prefixing it with "Assert" might be clearer, i.e. `container.AssertNoAspNetCoreRegistrations`?

Reply:
Yes! Was scratching my head here to think of a name. Let’s go with this.